code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -170,7 +170,7 @@ meteor_act
 	if(!affecting)
 		return //should be prevented by attacked_with_item() but for sanity.
 
-	visible_message("<span class='danger'>[src] has been [I.attack_verb.len? pick(I.attack_verb) : "attacked"] in the [affecting.name] with [I.name] by [user]!</span>")
+	visible_message("<span class='danger'>[src] has been [(I.attack_verb && I.attack_verb.len) ? pick(I.attack_verb) : "attacked"] in the [affecting.name] with [I.name] by [user]!</span>")

```

Comment:
alternately: `[LAZYLEN(I.attack_verb) ? pick(I.attack_verb) : "attacked"]`
There might be a macro for this (LAZYPICK), not sure if it was brought over.